README.md

```diff
@@ -10,3 +10,9 @@ Please download, unzip, and copy the corresponding asset folder for your platfor
 * Android - `/android/bin/release/`
 * Win32 - The win32 binaries are not included in the zip assets. They are available on download on [nuget](https://www.nuget.org/packages/Microsoft.PlayFab.PlayFabParty.Cpp.Windows).
 * Linux - `/x64/Release/libparty.so`
+
+## Troubleshooting
+
+### WSL Compatibility
```

Comment:
I would place the "WSL Compatibility" section outside (above) the Troubleshooting section (and remove the troubleshooting section for now), as this is technically not a troubleshooting-related note, but rather a description of the product.